playground/backend/configs/SDK_GO.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compile_cmd": "go",
+  "run_cmd": "",
```

Comment:
we don't need the run command here, right? we obtain a natively executable file, so we just run the output file?

Reply:
Yes, we don't need the run command, we just run the file that got after compilation.